src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -198,7 +216,11 @@ export class MeetingManager implements AudioVideoObserver {
     configuration: MeetingSessionConfiguration,
     deviceLabels: DeviceLabels | DeviceLabelTrigger = DeviceLabels.AudioAndVideo,
   ): Promise<any> {
-    const logger = this.createLogger(configuration);
+    if (this.videoDownlinkBandwidthPolicy) {
+      configuration.videoDownlinkBandwidthPolicy = this.videoDownlinkBandwidthPolicy;
+    }
+
+    const logger = this.logger ? this.logger : this.createLogger(configuration);
```

Comment:
Can this one be the first line as before? Not sure why we need to switch this after videoDownLinkBandwidthPolicy

Reply:
Yes, will update

Reply:
The reason is for better readability.
```
    if (this.videoDownlinkBandwidthPolicy) {
      configuration.videoDownlinkBandwidthPolicy = this.videoDownlinkBandwidthPolicy;
    }
```
This code is not related to the logger. Inserting it behind logger breaks the logic.

Reply:
what logic?